src/test/java/com/wikia/webdriver/testcases/adstests/TestAmazonAds.java

```diff
@@ -1,57 +1,56 @@
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
-import com.wikia.webdriver.common.core.url.Page;
 import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
 import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsAmazonObject;
 import org.testng.annotations.Test;
 
+@Test(groups = "AmazonAds")
 public class TestAmazonAds extends TemplateNoFirstLoad {
-  private static final String A9_TEST_LINE_ITEM = "4397742201";
-  private static final Page TEST_PAGE = new Page("project43", "SyntheticTests/Amazon");
-
-  private static final String[] DESKTOP_SLOTS = {
-          AdsContent.TOP_LB,
-          AdsContent.MEDREC
-  };
-
-  private static final String[] MOBILE_SLOTS = {
-          AdsContent.MOBILE_TOP_LB,
-          AdsContent.MOBILE_AD_IN_CONTENT,
-          AdsContent.MOBILE_PREFOOTER
-  };
 
   @NetworkTrafficDump
-  @Test(
-      dataProviderClass = AdsDataProvider.class,
-      groups = {"AmazonAds", "AmazonAdsDesktop"}
-  )
-  public void amazonAdsDesktop() {
-    checkAmazonSlots(DESKTOP_SLOTS);
+  @Test(groups = "AmazonAdsDesktop")
+  public void testAmazonDisplayAdsDesktop() {
+    checkAmazonSlots(AdsAmazonObject.DESKTOP_SLOTS);
   }
 
+  @Test(groups = "AmazonAdsMobile")
   @NetworkTrafficDump
-  @Test(
-      dataProviderClass = AdsDataProvider.class,
-      groups = {"AmazonAds", "AmazonAdsMobile"}
-  )
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
-  public void amazonAdsMobile() {
-    checkAmazonSlots(MOBILE_SLOTS);
+  public void testAmazonDisplayAdsMobile() {
+    checkAmazonSlots(AdsAmazonObject.MOBILE_SLOTS);
+  }
+
+  @Test(groups = "AmazonAdsDesktop")
+  @NetworkTrafficDump(useMITM = true)
+  public void testAmazonVideoAdsDesktop() {
+    testAmazonVideo();
+  }
+
+  @Test(groups = "AmazonAdsMobile")
+  @NetworkTrafficDump(useMITM = true)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void testAmazonVideoAdsMobile() {
+    testAmazonVideo();
   }
 
   private void checkAmazonSlots(String[] slots) {
     networkTrafficInterceptor.startIntercepting();
-    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, TEST_PAGE.getUrl());
+    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, AdsDataProvider.PAGE_A9_DISPLAY.getUrl());
     amazonAds.runA9DebugMode();
 
     for (String slotName : slots) {
-      amazonAds.verifyTestBidAdInSlot(slotName, A9_TEST_LINE_ITEM);
+      amazonAds.verifyTestBidAdInSlot(slotName, AdsAmazonObject.A9_TEST_LINE_ITEM);
     }
   }
+
+  private void testAmazonVideo() {
+    networkTrafficInterceptor.startIntercepting();
+    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, AdsDataProvider.PAGE_FV.getUrl(AdsAmazonObject.A9_VIDEO_DEBUG_MODE));
+    amazonAds.wait.forSuccessfulResponseByUrlPattern(networkTrafficInterceptor, AdsAmazonObject.A9_VIDEO_DEBUG_BID_PATTERN);
```

Comment:
I'm not sure if I understand it correctly - it checks whether we requested bid only, right? Wouldn't it be better to check whether we passed bid parameter to VAST request *and* check whether ad is playing? You can reuse `waitForAdPlaying`

https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsJWPlayerObject.java#L37